src/redis.c

```diff
@@ -61,8 +61,19 @@ redis_meta_get_reply(Meta m)
     int ret;
     redisReply *reply;
 
-    if ((ret = redisGetReply(m->c, (void *)&reply)) == REDIS_OK)
+    if ((ret = redisGetReply(m->c, (void *)&reply)) == REDIS_OK) {
         m->reply = reply;
+    } else {
+        switch (m->c->err) {
+        case REDIS_ERR_IO: /* I/O error, sets errno. */
+            logger_log("%s %d: %s %s",
+                       __FILE__, __LINE__, m->c->errstr, strerror(errno));
+            break;
+        default:
+            logger_log("%s %d: %s %s", __FILE__, __LINE__, m->c->errstr);
```

Comment:
gcc didn't yell at you for this?

Reply:
I forgot to rebase it upon that branch.
Good that we have you :) Thanks.